src/locales/fr.json

```diff
@@ -91,7 +92,8 @@
   "InstalledStep": {
     "title": "C'est presque fini !",
     "descriptionStart": "Cliquez sur l'icône",
-    "descriptionEnd": "en haut à droite de votre écran pour vous connecter à l'aide de votre addresse **%{address}** et de votre mot de passe Cozy."
+    "descriptionEnd": "en haut à droite de votre écran pour vous connecter à l'aide de votre addresse **%{address}** et de votre mot de passe Cozy.",
```

Comment:
in English we have a space at the start of the string, should we add it in french too?

Reply:
The spaces are already in the JSX file so it is not necessary, I've removed them in a subsequent commit.